homebrew_challenge/src/App.jsx

```diff
@@ -111,12 +111,12 @@ const App = () => {
           </Row>
           <Row className="pt-3 d-none d-sm-flex justify-content-center align-items-center">
             <Col>
-              <SpikeAlert />
+              <InfoBar/>
             </Col>
           </Row>
-          <Row className="pt-3 d-none d-sm-flex justify-content-center align-items-center">
+          <Row className="pt-3 d-sm-flex justify-content-center align-items-center">
```

Comment:
What do you want the behaviour to be for xs? If it is to be the same for all sizes, then either bin the d-sm-flex or change it to d-flex

Reply:
removed d-sm-flex

Reply:
Are you sure ? :)

Reply:
Never mind - GitHub annoyingly doesn't show the latest changes by default